multinode/types.go

```diff
@@ -58,8 +58,8 @@ type RPCClient[
 	SubscribeToHeads(ctx context.Context) (<-chan HEAD, Subscription, error)
 	// SubscribeToFinalizedHeads - returns channel and subscription for finalized heads.
 	SubscribeToFinalizedHeads(ctx context.Context) (<-chan HEAD, Subscription, error)
-	// Ping - returns error if RPC is not reachable
-	Ping(context.Context) error
+	// PingClientVersion - returns error if RPC is not reachable
```

Comment:
nit: Is just ClientVersion() a better name?
It is a method of the RPCClient, and just returns a client version.